pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -237,6 +237,12 @@ type PodSecurityPolicy struct {
 	// +kubebuilder:default=medium
 	// +optional
 	Severity string `json:"severity,omitempty"`
+	// ValidationFailureAction indicates the action to take when a pod creation fails to validate.
+	// For more info https://kyverno.io/docs/writing-policies/validate/#validation-failure-action
+	// +kubebuilder:validation:Enum=Enforce;Audit
+	// +kubebuilder:default=Audit
+	// +optional
+	ValidationFailureAction string `json:"validationFailureAction,omitempty"`
```

Comment:
looks confused to me. For Severity = high, it also prevents pod from creation.

Reply:
nop, I think that behavior is deprecated, `Severtiy` only affect [PolicyResultSeverity](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html#wgpolicyk8s.io/v1beta1.PolicyResultSeverity) on PolicyReport during my validation.